tutorials/VI_in_BM.ipynb

```diff
@@ -0,0 +1,2009 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        vi.world.initialize_value(guide)
</pre><p> </p><p>We will need to update this now that <code>world</code>  is no longer an attribute of <code>VariationalInfer</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>